Ska/DBI/tests/test_dbi.py

```diff
@@ -38,6 +39,12 @@ def teardown_class(cls):
         cls.db.cursor.close()
         cls.db.conn.close()
 
+    # Run an empty test to get the warning for no file (if running on a file)
+    # Ignore that warning in these tests.
+    @pytest.mark.filterwarnings("ignore:Database file missing")
```

Comment:
Here I'm just throwing away the new warnings so they don't show up in the pytest output.  If this warning idea is doable, do we want explicit testing of the warning?  If we care, I'd also need a new test that would make a database (check for warning) and then use a database (check there is no warning).